src/test/java/app/bpartners/api/unit/service/AccountRefreshServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package app.bpartners.api.unit.service;
+
+import static app.bpartners.api.model.BankConnection.BankConnectionStatus.OK;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+import app.bpartners.api.model.Account;
+import app.bpartners.api.model.User;
+import app.bpartners.api.repository.bridge.BridgeApi;
+import app.bpartners.api.repository.bridge.model.User.CreateBridgeUser;
+import app.bpartners.api.service.AccountRefreshService;
+import app.bpartners.api.service.AccountService;
+import app.bpartners.api.service.UserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AccountRefreshServiceTest {
+  UserService userServiceMock;
+  AccountService accountServiceMock;
+  BridgeApi bridgeApiMock;
+  AccountRefreshService subject;
+
+  @BeforeEach
+  void setUp() {
```

Comment:
Generally, you do not need to setUp these mocks before each method, and instead, you could just set its value directly in the class attributes. For eg, directly set AccountRefreshService subject  = new AccountRefreshService(...);